### PR TITLE
Expose $properties table for views in Hive connector

### LINKED ISSUE #8805: Expose lastddltime of hive views
Using trino there doesn't seem to be way to see the last time a Hive view was modified. Hive tables show transient_lastddltime in $properties. Perhaps Hive views should also have their own $properties ? Or alternatively add this value in information_schema.views

Having this would allow to answer the Q: what views were changed in the last 24hrs

### PR DESCRIPTION
**Pull request by ebyhr**
Fixes #8805

### CODE REVIEW

**Review by hashhar — COMMENTED** [review]
The implementation looks good to me but I'm not sure why the restriction existed initially.
I'll defer to @kokosing or @electrum for this.

File: testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    @Test(groups = HIVE_VIEWS)
+    public void testHiveViewPropertiesTable()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_hive_view_properties_base");
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_hive_view_properties");
+        onTrino().executeQuery("CREATE TABLE test_hive_view_properties_base (col INT)");
+        onHive().executeQuery("CREATE VIEW test_hive_view_properties AS SELECT * FROM test_hive_view_properties_base");
+
+        assertThat(query("SELECT * FROM \"test_hive_view_properties$properties\""))
+                .hasRowsCount(1);
```
Comment (hashhar) [inline testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java:51]: Do all Hive distributions only set just the timestamp as the property? Maybe the assertion needs to be more explicit?

File: testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    @Test(groups = HIVE_VIEWS)
+    public void testHiveViewPropertiesTable()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_hive_view_properties_base");
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_hive_view_properties");
+        onTrino().executeQuery("CREATE TABLE test_hive_view_properties_base (col INT)");
+        onHive().executeQuery("CREATE VIEW test_hive_view_properties AS SELECT * FROM test_hive_view_properties_base");
+
+        assertThat(query("SELECT * FROM \"test_hive_view_properties$properties\""))
```
Comment (kokosing) [inline testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java:50]: Can we verify the content of properties table?

**Comment by ebyhr** [comment]
The properties aren't equal among the current testing distributions. I would fix assertions, but it will be partial verification. 

ghcr.io/trinodb/testing/hdp3.1-hive
```bash
trino> SELECT * FROM hive.default."v8805$properties";
 bucketing_version | transient_lastddltime
-------------------+-----------------------
 2                 | 1628499683
```

ghcr.io/trinodb/testing/cdh5.15-hive
```bash
trino> SELECT * FROM hive.default."v8805$properties";
 transient_lastddltime
-----------------------
 1628506166
```

**Comment by ebyhr** [comment]
@kokosing @hashhar Updated tests:
- Add an assertion with `SHOW COLUMNS FROM` 
- Verify properties with `SHOW TBLPROPERTIES` in Hive result

**Review by hashhar — APPROVED** [review]
LGTM % ci with hive tests.

The new tests look much better. Thanks.
@ebyhr Can you please push an empty commit to get all hive tests to run?

File: testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_hive_view_properties");
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_hive_view_properties_base");
+    }
+
+    private static List<String> getTablePropertiesOnHive(String tableName)
+            throws SQLException
+    {
+        ImmutableList.Builder<String> properties = ImmutableList.builder();
+        try (Statement statement = onHive().getConnection().createStatement();
```
Comment (kokosing) [inline testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java:86]: `onHive().executeQuery("SHOW TBLPROPERTIES " + tableName).rows()`?

**Review by kokosing — APPROVED** [review]

File: testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java (hunk 3 above)
Comment (ebyhr) [reply testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java]: Thanks, changed to use the suggested method.

**Pull request merged**